pkg/policy/resolve.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+// Policy is a structure which contains the resolved policy across all layers
+// (L3, L4, and L7).
+type Policy struct {
```

Comment:
@aanm suggested we call this an `EndpointPolicy`, since it is currently the policy for a given endpoint. Thoughts?